pubtools/pulplib/_impl/model/maintenance.py

```diff
@@ -0,0 +1,193 @@
+import logging
+import datetime
+import json
+import os
+
+import jsonschema
+
+from pubtools.pulplib._impl import compat_attr as attr
+from ..schema import load_schema
+from .common import InvalidDataException
+
+
+LOG = logging.getLogger("pubtools.pulplib")
+
+
+USER = os.environ.get("USER")
+HOSTNAME = os.environ.get("HOSTNAME")
+
+
+def iso_time_now():
+    return datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
+@attr.s(kw_only=True, frozen=True)
+class MaintenanceEntry(object):
+    """Details about the maintenance status of a specific repository."""
+
+    repository_id = attr.ib(type=str)
```

Comment:
Sorry, I made a mistake here.  In Task class, the same kind of attribute was called "repo_id".  So, even though it goes against the guideline of avoiding abbreviations, consistency is more important and we should in fact call this:

```suggestion
    repo_id = attr.ib(type=str)
```